msteams-platform/toolkit/provision.md

```diff
@@ -1,621 +1,435 @@
 
-* Ensure that the folder structure and file name remain unchanged. The tool may append new content to the existing files when you add more resources or capabilities to your project.
-* Ensure that the name of auto-generated parameters and its property names remain unhanged. The auto-generated parameters may be used when you add more resources or capabilities to your project.
-* Ensure that the output of auto-generated ARM template is unchanged. You can add more outputs to ARM template. The output is `.fx\states\state.{env}.json` and can be used in other features, such as deploy and validate manifest files.
+> * Ensure not to share the same Azure AD app in multiple environments.
 
-### Customize Teams app
+#### Use an existing Azure AD app for your bot
 
-You can customize your bot or the Teams app by adding configuration snippets to use an Azure AD app created for your Teams app.
-Perform in the following ways to customize the Teams app:
+You can follow below steps to add environment variables to the .env files to use an Azure AD app created for your Teams app. If you don't have an Azure AD app for your bot yet or you already have one but don't know where to find the correct values, see [Use existing Azure AD app in TeamsFx project](use-existing-aad-app.md).
 
-* [Use an existing Azure AD app for your Teams app](#use-an-existing-azure-ad-app-for-your-teams-app)
-* [Use an existing Azure AD app for your bot](#use-an-existing-azure-ad-app-for-your-bot)
+1. Open `teamsapp.yml` and find the `botAadApp/create` action.
 
-#### Use an existing Azure AD app for your Teams app
+1. Find the environment variable names that store information for Azure AD app in the `writeToEnvironmentFile` property. Below is the default `writeToEnvironmentFile` definition if you create projects using Teams Toolkit:
 
-You can add the following configuration snippet to `.fx\configs\config.{env}.json` file to use an Azure AD app created for your Teams app. If you don't have an Azure AD app yet or you already have one but don't know where to find the correct value, see [how to use existing Azure AD app in TeamsFx project](https://github.com/OfficeDev/TeamsFx/blob/dev/docs/fx-core/using-existing-aad.md):
-
-```json
-"$schema": "https://aka.ms/teamsfx-env-config-schema",
-"description": "...",
-"manifest": {
-  ...
-},
-// Add code below. Note you need to replace the placeholders with real values.
-"auth": {
-    "clientId": "<your Azure AD app client id>",
-    "clientSecret": "{{$env.ENV_NAME_THAT_STORES_YOUR_SECRET}}",
-    "objectId": "<your Azure AD app object id>",
-    "accessAsUserScopeId": "<id of the access_as_user scope>"
-}
+```yml
+ writeToEnvironmentFile:
+   botId: BOT_ID
+   botPassword: SECRET_BOT_PASSWORD
 ```
 
-After adding the snippet, add your client secret to the related environment variable for Teams Toolkit to resolve the actual client secret during provision.
+1. Add values for each environment variable from step 2.
```

Comment:
The numbering is wrong in the generated content

Reply:
![image](https://github.com/MicrosoftDocs/msteams-docs/assets/16605901/5d098c80-b9a1-46ef-b8b7-a11335ae1a95)

Reply:
Updated.